Slack&whatsapp-notifier.py

```diff
@@ -154,9 +130,7 @@ def main():
                 result = execute_query(mycursor, query,t)

```

Comment:
1) process_failed_transactions, this function is checking for the threshold reached, not specifically for failed transaction.
2) make message_extraction() take the mode i.e SLACK or WHATSAPP and fetch required template
3) 'send_whatsapp_message("MULTIPLE FAILED TRANSACTIONS ERROR!", time.localtime().tm_hour, time.localtime().tm_min)', here use template instead of static message
4) add credentials to the env file in send_slack_message
5) remove unnecessary comments from main function
6) move all the prints to a log file
7) why are we doing while(True)?